0000-lower-minimum-device-onboarding-fees

```diff
@@ -0,0 +1,113 @@
+# HIP XX: Lower Minimum Device Onboarding Fees
```

Comment:
If you're going to resubmit this HIP (HIP 81), you need to address WHY you're resubmitting HIP 81 and what you changed from the previous time. 

Be sure to address that although HIP 81 did not pass, we are resubmitting the HIP with X changes and believe it is still necessary because of A, B, C problems that exist in the network.

Reply:
Adding why it is believed that the slight modifications to HIP81 should allow the HIP to pass.  I am of the position that the problems being addressed are already part of the content.  They are addressing exactly the same issue.